plugins/vpc-shared-eni/network/bridge_linux.go

```diff
@@ -485,29 +517,54 @@ func (nb *BridgeBuilder) createBridge(
 	} else {
 		// In layer3 configuration, the IP address and default route remain on the shared ENI.
 		// IP datagrams are routed between the bridge and the shared ENI.
+		if vpc.ListContainsIPv4Address(ipAddresses) {
+			// Bridge proxies ARP requests originating from veth pairs to the VPC.
+			log.Infof("Enabling IPv4 proxy ARP on %s.", bridgeName)
+			err = ipcfg.SetIPv4ProxyARP(bridgeName, 1)
+			if err != nil {
+				log.Errorf("Failed to enable IPv4 proxy ARP on %s: %v.", bridgeName, err)
+				return 0, err
+			}
 
-		// Bridge proxies ARP requests originating from veth pairs to the VPC.
-		log.Infof("Enabling IPv4 proxy ARP on %s.", bridgeName)
-		err = ipcfg.SetIPv4ProxyARP(bridgeName, 1)
-		if err != nil {
-			log.Errorf("Failed to enable IPv4 proxy ARP on %s: %v.", bridgeName, err)
-			return 0, err
-		}
+			// Enable IPv4 forwarding on the bridge and shared ENI, so that IP datagrams can be
+			// routed between them.
+			log.Infof("Enabling IPv4 forwarding on %s.", bridgeName)
+			err = ipcfg.SetIPv4Forwarding(bridgeName, 1)
+			if err != nil {
+				log.Errorf("Failed to enable IPv4 forwarding on %s: %v.", bridgeName, err)
+				return 0, err
+			}
 
-		// Enable IPv4 forwarding on the bridge and shared ENI, so that IP datagrams can be
-		// routed between them.
-		log.Infof("Enabling IPv4 forwarding on %s.", bridgeName)
-		err = ipcfg.SetIPv4Forwarding(bridgeName, 1)
-		if err != nil {
-			log.Errorf("Failed to enable IPv4 forwarding on %s: %v.", bridgeName, err)
-			return 0, err
+			log.Infof("Enabling IPv4 forwarding on %s.", sharedENI.GetLinkName())
+			err = ipcfg.SetIPv4Forwarding(sharedENI.GetLinkName(), 1)
+			if err != nil {
+				log.Errorf("Failed to enable IPv4 forwarding on %s: %v.", sharedENI.GetLinkName(), err)
+				return 0, err
+			}
 		}
 
-		log.Infof("Enabling IPv4 forwarding on %s.", sharedENI.GetLinkName())
-		err = ipcfg.SetIPv4Forwarding(sharedENI.GetLinkName(), 1)
-		if err != nil {
-			log.Errorf("Failed to enable IPv4 forwarding on %s: %v.", sharedENI.GetLinkName(), err)
-			return 0, err
+		if vpc.ListContainsIPv6Address(ipAddresses) {
+			// Enable IPv6 forwarding on all interfaces in k8s-netns, this seems to be required for IPv6
```

Comment:
Please remove the mention of k8s-netns. Simply "Eanble IPv6 forwarding on all interfaces.".